.eslintrc

```diff
@@ -85,7 +85,6 @@
     "react/jsx-key": ["error"],
     "react/jsx-max-depth":["error", { "max": 5}],
     "react/jsx-max-props-per-line": ["error", {"maximum": 3, "when": "multiline" }],
-    "react/jsx-no-bind": ["error"],
```

Comment:
Sometimes, it doesn't make sense to create an extra function. example: 

`() => setShowSuggestions(true)`

I would prefer a simple version instead of

```
handleSetShowSuggestons = () => {
 setShowSuggestions(true)
}
```

Reply:
Could you give me a real example? Usually `useCallback` worth, others do not.

Reply:
https://github.com/verdaccio/ui/blob/5592f107a1296a90ae42b76606c8d24955be804f/src/components/LanguageSwitch/LanguageSwitch.tsx#L92

Reply:
I see, well, in that case is ok. But just few lines below you might improve the performance of your component memoazing this

```
        options={languages.map(language => {
          const { icon, translation } = getTranslatedCurrentLanguageDetails(t, language);
          return {
            language,
            icon,
            translation,
          };
        })}
```
Languages hardly will change  so this is an scenario where makes totally sense using `useMemo` .

Reply:
I'd restore the rule, if sometimes does not make sense, you always can disable the rule for such line. In the worse case, you can customize it https://github.com/yannickcr/eslint-plugin-react/blob/master/docs/rules/jsx-no-bind.md#rule-options and make it more flexible.